### DISCUSSION TITLE
WPF sessions at Build 2024

### DISCUSSION THREAD

**Discussion by miloush** [dotnet/wpf#9149] [General]
Live stream: https://build.microsoft.com/

[Navigating Win32 App Development with WinUI and WPF](https://build.microsoft.com/en-US/sessions/11626139-a9d0-4f8c-b664-3f3436cea50a) (45 min)
| Zone | Day | Time |
|--|--|--|
| PDT | Wednesday, May 22 | 4:45 PM |
| CEST | Thursday, May 23 | 01:45 |
| IST | Thursday, May 23 | 05:15 |
| CST | Thursday, May 23 | 07:45 |

[How to create superior experiences with WinUI and WPF](https://build.microsoft.com/en-US/sessions/5c7afbc3-1e4c-41bb-b883-b66f78b46be7) (45 min)
| Zone | Day | Time |
|--|--|--|
| PDT | Thursday, May 23 | 3:00 PM |
| CEST | Friday, May 24 | 00:00 |
| IST | Friday, May 24 | 06:00 |
| CST | Friday, May 24 | 03:30 |

**Comment by lindexi** [comment]
Awesome!

**Comment by pchaurasia14** [comment]
Thanks @miloush for sharing these!
